buildconfig/configure-5.9.0.sh

```diff
@@ -13,6 +13,7 @@ make -j$CORE_COUNT > /dev/null 2>&1 || make -j$CORE_COUNT
 make test > /dev/null 2>&1 || make test
 make install > /dev/null 2>&1 || make install
 cd ..
+sed -i -e 's/"-lgds"/"-lfbclient"/' ../qtbase/src/plugins/sqldrivers/configure.json
```

Comment:
That line should get removed, it's the last thing left, all other builds were fine.